src/content/guides/using-persistent-volumes-with-csi-driver-on-aws/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Using Persistent Volumes with EBS CSI driver on AWS"
+description: Tutorial on how to use dynamically provisioned Persistent Volumes with EBS CSI driver on a cluster running on Amazon Web Services"
+type: page
+weight: 50
+tags: ["tutorial"]
+user_questions:
+  - How do I install the EBS CSI driver?
+  - How do I provision an EBS instance on AWS?
+  - How do I test the EBS CSI storage class?
+  - How do I use EBS CSI Volumes?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-firecracker
+---
+
+# Using Persistent Volumes on AWS with Container Storage Interface (CSI)
+
+CSI has been promoted to GA in Kubernetes v1.13 and is becoming the standard to replace the current Kubernetes `in-tree` storage plugin to handle Volumes for different provider.
```

Comment:
Ideally we introduce non-common abbreviations (or those where we are not certain if everybody understands them) in text by giving the long form first and the abbreviation in parentheses.

```suggestion
The Container Storage Interface (CSI) has been promoted to general availability (GA) in Kubernetes v1.13 and is becoming the standard to replace the current Kubernetes `in-tree` storage plugin to handle volumes for different providers.
```